blocks/header/header.css

```diff
@@ -170,6 +177,17 @@ header nav.nav-bottom .menu-level-1-item > a {
   justify-content: space-between;
 }
 
+header nav.nav-middle h5 > a:hover {
+  line-height: 1;
+  text-decoration: none;
+  background-color: var(--secondary);
```

Comment:
All styles other than color and background-color is already there in header nav.nav-middle h5 > a {.
So, i think no need to add here.

Reply:
updated.